scripts/partition_manager.py

```diff
@@ -938,7 +940,7 @@ def expect_addr_size(td, name, expected_address, expected_size):
     if expected_address:
         assert td[name]['address'] == expected_address, \
             "Address of {} was {}, expected {}.\ntd:{}".format(name, td[name]['address'], expected_address, pformat(td))
-    if expected_size and expected_address:
+    if expected_size and expected_address and 'end_address' in td[name]:
```

Comment:
I'm skeptical of this. end_address should always be present, so if it's not, it's a bug :P

Reply:
@oyvindronningstad The test vectors in master does not have 'end_address' and this function asserts. That is why I changed this line.

Reply:
OK, let's keep it so it passes on master. Will fix later.